pyproject.toml

```diff
@@ -62,9 +62,9 @@ pytest-cov = "^2.7"
 pytest-randomly = "^3.2"
 pytest-mypy-plugins = "^1.2"
 
-sphinx = "^2.3"
+sphinx = "^3.0"
```

Comment:
This also requires these changes: https://github.com/wemake-services/wemake-python-styleguide/pull/1439

Because `sphinx@3.0` does not work with `m2r`